doc/release-notes-12677.md

```diff
@@ -0,0 +1,7 @@
+Notable changes
+===============
+
+Updated RPCs
+------------
+
+- `listunspent` now includes `ancestorcount`, `ancestorsize`, and `ancestorfees` in some cases.
```

Comment:
suggestion: s/in some cases/for each transaction output that is still in the mempool/